lib/dradis/plugins/projects/upload/v1/template.rb

```diff
@@ -38,6 +38,10 @@ def post_initialize(args={})
 
       private
 
+      def create_comments(commentable, xml_comments)
```

Comment:
probably add a comment that this is overridden in V2?

Reply:
✅